src/libs/DoInteractionTask/index.ts

```diff
@@ -0,0 +1,6 @@
+function doInteractionTask(callback: () => void) {
+    callback();
+    return null;
```

Comment:
NAB: Is this returning null for explicit clarity or does it serve any other purpose as well?

Reply:
I saw that. By default it returns undefined anyway if we dont add any return statement. Returning null does not appear to serve any additional purpose. That's why I asked.